lnwallet/script_utils.go

```diff
@@ -808,6 +808,32 @@ func htlcSpendSuccess(signer Signer, signDesc *SignDescriptor,
 	return witnessStack, nil
 }
 
+// HtlcSpendSuccess exposes the public witness generation function for spending
```

Comment:
Why not just publicly export the private function about this?

Reply:
sequence, version and sig hashes are already populated by the caller. The private method also doesn't exactly match the WitnessGenerationFunc signature

Reply:
Done!